openmrs/src/main/java/org/motechproject/openmrs/domain/ProgramEnrollmentListResult.java

```diff
@@ -14,6 +14,12 @@
         return results;
     }
 
+    public ProgramEnrollment getFirstObject() { return results.get(0); }
```

Comment:
what if ProgramEnrollmentList length is 0 ? I suppose that will be exception here